CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 2.0.0
+
+## Dependency Update
+
+The following packages are now specified as peer dependencies:
+
+* react
+* react-dom
+* flux
+
+The following packages have been moved from `devDependencies` to `dependencies`:
+
+* `i18n-js`
+* `immutable`
+* `react-router`
+* `react-transition-group`
```

Comment:
Is this what you had in mind
https://github.com/Sage/carbon/blob/efe7142e41c37cbdca0222179f1441a1b08ce7cc/CHANGELOG.md#upgrading-a-project-that-uses-carbon